homeassistant/components/overkiz/executor.py

```diff
@@ -71,3 +72,12 @@ async def async_execute_command(self, command_name: str, *args: Any):
     async def async_cancel_command(self, exec_id: str):
         """Cancel device command in async context."""
         await self.coordinator.client.cancel_command(exec_id)
+
+    def get_gateway_id(self):
+        """
+        Retrieve gateway id from device url.
+
+        device URL (<protocol>://<gatewayId>/<deviceAddress>[#<subsystemId>])
```

Comment:
What does this url look like in production?

Reply:
`internal://1234-5678-1234/pod/0`
`io://1234-5678-1234/12341234#2`